src/main/java/com/ss/mqtt/broker/handler/publish/out/PublishOutHandler.java

```diff
@@ -9,5 +9,5 @@
  */
 public interface PublishOutHandler {
 
-    void handle(@NotNull PublishInPacket packet, @NotNull Subscriber subscriber);
+    boolean handle(@NotNull PublishInPacket packet, @NotNull Subscriber subscriber);
```

Comment:
why do we need to have boolean here?

Reply:
Because we should answer with appropriate PublishAckReasonCode